tests/test_algorithm.py

```diff
@@ -3194,3 +3195,79 @@ def transactions_for_date(date):
                 'order_id': None,  # Auto-close txns emit Nones for order_id.
             },
         )
+
+
+class TestOrderAfterDelist(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.env = TradingEnvironment()
+
+        cls.days = cls.env.days_in_range(
+            start=pd.Timestamp("2016-01-05", tz='UTC'),
+            end=pd.Timestamp("2016-01-15", tz='UTC')
+        )
+
+        # asset goes from 1/5 to 1/6.
+        # asset liquidate date is 1/11.
+        cls.env.write_data(equities_data={
+            1: {
+                'start_date': cls.days[0],
+                'end_date': cls.days[1],
+                'auto_close_date': cls.days[4],
+                'symbol': "ASSET1"
+            }
+        })
+
+        cls.data_portal = FakeDataPortal(cls.env)
+
+    def test_order_in_quiet_period(self):
+        algo_code = dedent("""
+        from zipline.api import (
+            sid,
+            order,
+            order_value,
+            order_percent,
+            order_target,
+            order_target_percent,
+            order_target_value
+        )
+
+        def initialize(context):
+            pass
+
+        def handle_data(context, data):
+            order(sid(1), 1)
+            order_value(sid(1), 100)
+            order_percent(sid(1), 0.5)
+            order_target(sid(1), 50)
+            order_target_percent(sid(1), 0.5)
+            order_target_value(sid(1), 50)
+        """)
+
+        # run algo from 1/7 to 1/8
+        algo = TradingAlgorithm(
+            script=algo_code,
+            env=self.env,
+            sim_params=SimulationParameters(
+                period_start=pd.Timestamp("2016-01-07", tz='UTC'),
+                period_end=pd.Timestamp("2016-01-07", tz='UTC'),
+                env=self.env,
+                data_frequency="minute"
+            )
+        )
+
+        log_catcher = TestHandler()
```

Comment:
We should use [`make_test_handler`] (https://github.com/quantopian/zipline/blob/6b3560ade8ac9294fd4b5db71948881b6cc673ab/zipline/testing/core.py#L905-L905), so the handler doesn't leak stack frames.